neuralprophet/plot_forecast_matplotlib.py

```diff
@@ -131,6 +138,17 @@ def plot(
                         alpha=0.2,
                     )
 
+    # Plot any conformal prediction intervals
```

Comment:
for the default plotting method I suggest we only plot one uncertainty type at once, as it may be confusing to most users to see multiple uncertainty types at once.